client/src/components/ElementEditor/InlineEditForm.js

```diff
@@ -1,9 +1,57 @@
 import React, { PureComponent, PropTypes } from 'react';
+import { connect } from 'react-redux';
 import classnames from 'classnames';
 import FormBuilderLoader from 'containers/FormBuilderLoader/FormBuilderLoader';
 import { loadElementSchemaValue } from 'state/editor/loadElementSchemaValue';
+import { bindActionCreators } from 'redux';
+import { change } from 'redux-form/lib';
+import Config from 'lib/Config';
 
 class InlineEditForm extends PureComponent {
+  constructor(props) {
+    super(props);
+
+    const sectionKey = 'DNADesign\\Elemental\\Controllers\\ElementalAreaController';
+    const section = Config.getSection(sectionKey);
+    const formNameTemplate = section.form.elementForm.formNameTemplate;
```

Comment:
You can use `loadElementSchemaValue('formNameTemplate')` for this (see `client/src/state/editor/loadElementSchemaValue.js`)